### DISCUSSION TITLE
How the language/internationalization is handled in this template?

### DISCUSSION THREAD

**Discussion by maroshvarchola** [alleyinteractive/create-wordpress-plugin#351] [Q&A] [answered]
So far, I did not see any of internationalization techniques for plugin to work ok, is there some mechanism I did not see? Thanks for answer in advance!

**Comment by srtfisher** [comment, marked as answer]
We use internationalization from core itself: https://developer.wordpress.org/themes/classic-themes/functionality/internationalization/
